src/libs/ajax/methods/Methods.ts

```diff
@@ -47,6 +47,19 @@ export const Methods = (signal?: AbortSignal) => ({
     return res.json();
   },
 
+  getNamespacePermissions: async (namespace: string): Promise<MethodConfigACL> => {
```

Comment:
Just for me, could you clarify what a namespace is, and what the new name will be if there will be a change as part of the PR Adam is working on?
It looks to me like a namespace is the same thing as a workflow / modal but I think I am missing a critical nuance here sorry

Reply:
So it is similar to workspace namespace where one namespace can have many different workflows/methods inside a namespace. And similar to its workspace friend, a workflow/method namespace can have separate set of permissions (which are maintained in Agora and not Sam). 

In the new world, its new name based on mockups is going to be `collection`.